usage/app/lib/BadInputException.scala

```diff
@@ -1,3 +1,5 @@
 package lib
 
-class BadInputException(message: String) extends Exception
+class BadInputException(message: String) extends Exception {
+  override def getMessage: String = message
+}
```

Comment:
Would passing it straight through to `Exception` work?
```suggestion
class BadInputException(message: String) extends Exception(message)
```

Reply:
Almost certainly.